scripts/js/onboard/README.md

```diff
@@ -4,4 +4,4 @@ Simple Node JS script that onboard the parachain against a local target
 
 ## Requirements
 
-- yarn 1.22.x
+- Node 20+
```

Comment:
I like that yarn is being replaced with npm in the scripts.
Do we also want to remove `yarn` from `scripts/runtime-dev-upgrade.sh`, `scripts/runtime-upgrade.sh`?